test/utils/clean/clean.go

```diff
@@ -47,22 +58,10 @@ func Configs() error {
 	}
 
 	for _, ptpConfig := range ptpconfigList.Items {
-		if ptpConfig.Name == utils.PtpGrandMasterPolicyName || ptpConfig.Name == utils.PtpSlavePolicyName {
-			err = client.Client.PtpConfigs(utils.PtpLinuxDaemonNamespace).Delete(context.Background(), ptpConfig.Name, metav1.DeleteOptions{})
-			if err != nil {
-				return fmt.Errorf("clean.All: Failed to delete ptp config %s %v", ptpConfig.Name, err)
-			}
-		}
-	}
-	for i := 0; i < 20; i++ {
-		ptpconfigList, err = client.Client.PtpConfigs(utils.PtpLinuxDaemonNamespace).List(context.Background(), metav1.ListOptions{})
+		err = client.Client.PtpConfigs(utils.PtpLinuxDaemonNamespace).Delete(context.Background(), ptpConfig.Name, metav1.DeleteOptions{})
```

Comment:
The clean up should only clean up policy created by the test . Not the discovered (configured) PTP config

Reply:
fixed